target/linux/lantiq/files/arch/mips/boot/dts/lantiq/vr9_bt_homehub-v5a.dts

```diff
@@ -210,9 +215,17 @@
 		tx-internal-delay-ps = <1500>;
 		rx-internal-delay-ps = <1500>;
 		phy-handle = <&phy5>;
+
+		nvmem-cells = <&macaddr_caldata_110c 4>;
+		nvmem-cell-names = "mac-address";
 	};
 };
 
+&mei {
```

Comment:
Build fails on:
~~~
Error: /home/aleksander/workspace/openwrt/build_dir/target-mips_24kc_musl/linux-lantiq_xrx200/linux-6.6.74/arch/mips/boot/dts/lantiq/vr9_bt_homehub-v5a.dts:224.1-5 Label or path mei not found
FATAL ERROR: Syntax error parsing input tree
~~~